plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/statistics/DeltaLakeJsonFileStatistics.java

```diff
@@ -155,6 +171,12 @@ private Optional<Object> deserializeStatisticsValue(DeltaLakeColumnHandle column
     }
 
     private static Long readStatisticsTimestamp(String timestamp)
+    {
+        LocalDateTime localDateTime = LocalDateTime.parse(timestamp, PARTITION_TIMESTAMP_FORMATTER);
+        return localDateTime.toEpochSecond(UTC) * MICROSECONDS_PER_SECOND + localDateTime.getNano() / NANOSECONDS_PER_MICROSECOND;
+    }
+
+    private static Long readStatisticsTimestampWithZone(String timestamp)
```

Comment:
the diff is hard to read, since old method implementation gains new method name
rename first (in the prep commit renaming `timestampReader`)